.github/CODEOWNERS

```diff
@@ -3,3 +3,4 @@
 /cobalt/  # no owners
 /media/starboard/  # no owners
 /starboard/  # no owners
+/components/viz/service/display/starboard/  # no owners
```

Comment:
@kaidokert @briantting  we should probably consider just having a `**/starboard` and/or `**/cobalt` pattern here/in pre-commit. Nothing to do here though

Reply:
Yeah, that's not a bad idea, anything under `**/starboard/**` pattern is included. We should pick one or the other though ( cobalt or starboard ) and stick with it.

Reply:
Move this to https://github.com/youtube/cobalt/pull/4352, and feel free to create a PR to fix this. Otherwise, I will keep it right now.

Reply:
Sg—no need for you to do anything about this issue, the infra team can figure out the approach they want